apps/faucet/src/config/tokens.ts

```diff
@@ -7,12 +7,15 @@ type TokenConfigType = Record<string, string>;
 const tokens: Record<string, TokenConfigType> = {
   [PresetTypedChainId.AthenaOrbit]: {
     webbWETH: '0x915EbFAd9c4Af2F420782a5F03BD544f9a1FF1d1',
```

Comment:
Do we still support `webbWETH` on the faucet? If so, we should update the API to include the token address field. Currently, we only have the `typedChainId` and `walletAddress` fields (details [here](https://www.chatbotui.com/url)). How would the backend know which token to send to the user if we support multiple tokens on the same chain?

Reply:
No, we do not, I did not remove it just in-case it would break something in the UI

Reply:
You can remove it. We read and render these values dynamically based on the config. Therefore, if you remove it, that token will not appear on the UI.